public/app/features/migrate-to-cloud/onprem/ResourceDetailsModal.tsx

```diff
@@ -9,11 +9,28 @@ interface ResourceDetailsModalProps {
   onClose: () => void;
 }
 
+const resourceErrorMessage: { [key: string]: string } = {
+  DATASOURCE_NAME_CONFLICT: t('migrate-to-cloud.resource-details.error-messages.datasource-name-conflict', ''),
```

Comment:
I think this needs to be like this:
```suggestion
  DATASOURCE_NAME_CONFLICT: t('migrate-to-cloud.resource-details.error-messages.datasource-name-conflict', 'There is a data source with the same name in the target instance. Rename one of them and try again.'),
```

And then you do `make i18n-extract` to generate the locales files (it should generate one for en-US and a pseudo-locale.

Reply:
figured how to use them and updated, should have set this to draft beforehand 😅